src/main/scala/bubblewrap/PageParser.scala

```diff
@@ -45,8 +45,9 @@ trait ContentType {
 
   def isGzip(content: Content) = {
     (content.contentEncoding, content.contentType) match {
-      case (Some(encoding), _) => gzipVariants.exists(gz => gz.equals(encoding))
-      case (_, Some(cType)) => gzipVariants.exists(gz => gz.equals(cType))
+      case (Some(encoding), Some(cType)) => gzipVariants.contains(encoding) || gzipVariants.contains(cType)
```

Comment:
@paambaati Can we re-place this entire switch statements with the following snippet? 
```
content.contentEncoding.exists(gzipVariants.contains) || content.contentType.exists(gzipVariants.contains)
```

Reply:
@ashwanthkumar Fixed in 38bee72.